modulepath-tests/pom.xml

```diff
@@ -118,4 +118,25 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>parallelTest</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>${maven.surefire.plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <id>default-test</id>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```

Comment:
This looks a bit different than the profile we have in the EE repo's module. I think we need to consolidate the two, such as we need to make sure the tests in the OS module don't "phone home". The EE module also allows running the `QuickTest`s. Not sure that's needed, we have only `ModulePathTests` in that module.

Reply:
I'll check it and if needed I'll align the EE version to be more like this OS one.

Reply:
I found a simpler way how to handle the parallel test by redefining the `javaModuleArgs` property. I've sent the update on both OS and EE side. Please recheck.